Alignment/APEEstimation/plugins/ApeEstimator.cc

```diff
@@ -1087,23 +1087,23 @@ ApeEstimator::fillHitVariables(const TrajectoryMeasurement& i_meas, const edm::E
               { errorWithoutAPE = LocalError(errHitApe.xx() -lape.xx(), errHitApe.xy()- lape.xy(), errHitApe.yy()-lape.yy());
 
                }
-	 }
-	if(Strip){
+   }
+  if(Strip){
     if(!dynamic_cast<const StripTopology*>(&detUnit.type().topology())){hitParams.hitState = TrackStruct::invalid; return hitParams;}
      const StripGeomDetUnit * stripDet = (const StripGeomDetUnit*)(&detUnit);
      const LocalError& lape = stripDet->localAlignmentError();
            if (lape.valid())
               { errorWithoutAPE = LocalError(errHitApe.xx() -lape.xx(), errHitApe.xy()- lape.xy(), errHitApe.yy()-lape.yy());
-		}
-	}
+    }
+  }
 
 
   const LocalError& errHitWoApe = errorWithoutAPE;
   const LocalError& errTrk = tsos.localError().positionError();
   
   const StatePositionAndError2 positionAndError2Hit = this->positionAndError2(lPHit, errHitApe, hit);
   const StatePositionAndError2 positionAndError2HitWoApe = this->positionAndError2(lPHit, errHitWoApe, hit);
-	std::cout<<"errHitWoApe  " <<errHitWoApe<<"errHitApe   "<<errHitApe<<std::endl;
+  std::cout<<"errHitWoApe  " <<errHitWoApe<<"errHitApe   "<<errHitApe<<std::endl;
```

Comment:
@mteroerd as you are touching this: you have to use the MessageLogger instead of `std::cout`

Reply:
Done.